docs/dashboards/index.md

```diff
@@ -1,7 +1,9 @@
 title: Sematext Dashboards
 description: Sematext Cloud is a modern monitoring, log management, transaction tracing, real user monitoring, and synthetic monitoring system. Dashboards are a powerful feature for cohesive data analysis from multiple data sources such as logs, metrics, and events.
 
-Dashboards are your central location where you put everything together. Logs, application performance metrics, infrastructure inventory and processes, cluster data with Docker and Kubernetes metrics, and events that give you insight into what is happening with your system in real-time. Everything in one place. 
+Dashboards are your central location where you put everything together. Logs, application performance metrics, infrastructure inventory and processes, cluster data with Docker and Kubernetes metrics, and events that give you insight into what is happening with your system in real-time. Everything in one place for easy correlation and troubleshooting. 
```

Comment:
Can you put Synthetics (e.g. custom metric or core web vitals metric) on a top level dashboard together with a chart with logs or infra metrics?  If yes, this could be useful to mention or show eventually.... you know what I'm thinking... :)

Reply:
Added
In the example below, we have an Apache service endpoint and we are monitoring Core Web Vitals with [Synthetics App](https://sematext.com/docs/synthetics/), we are monitoring Apache service performance metrics such as CPU, Memory usage and Network Traffic with a [Monitoring App](https://sematext.com/docs/monitoring/) and we shipping logs to a Apache [Logs App](https://sematext.com/docs/logs/). You can put all the data coming from these Apps in a single dashboard to have everything in one place.
with a screenshot